src/index.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
```

Comment:
- [ ] Make sure to use recommended font-family type that is `Lato` or `Gill sans`.

![Screenshot from 2021-09-07 16-55-47](https://user-images.githubusercontent.com/22434660/132456147-213225c6-297b-4e70-8be3-f455fd06de48.png)

Reply:
Done